tests/baselines/reference/decoratorOnClass8.errors.txt

```diff
@@ -6,6 +7,7 @@ tests/cases/conformance/decorators/class/decoratorOnClass8.ts(3,1): error TS2322
     
     @dec()
     ~~~~~~
-!!! error TS2322: Type '(target: Function, paramIndex: number) => void' is not assignable to type '(target: typeof C) => void | typeof C'.
+!!! error TS1220: Invalid expression for class decorator.
+!!! error TS1220:   Supplied parameters do not match any signature of call target.

```

Comment:
I think the head message needs to be a bit more descriptive. Something like:

`Unable to resolve signature of {0} decorator when called as an expression.`